recipes/README.md

```diff
@@ -41,31 +41,34 @@ python delete_all_assets.py
 
 ## How to push pre-annotation to existing assets
 
-Check out [our recipe on how to import predictions](https://github.com/kili-technology/kili-playground/blob/master/recipes/import_predictions.ipynb) (run it [here](https://colab.research.google.com/github/kili-technology/kili-playground/blob/master/recipes/import_predictions.ipynb))
+Check out [our recipe on how to import predictions](https://github.com/kili-technology/kili-python-sdk/blob/master/recipes/import_predictions.ipynb) (run it [here](https://colab.research.google.com/github/kili-technology/kili-python-sdk/blob/master/recipes/import_predictions.ipynb))
```

Comment:
here and below, relative paths should work too

Reply:
won't they work when the repo will be renamed?